cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -392,11 +393,22 @@ public void initialize(MasterEnvironmentContext context) throws IOException, Ill
                namespace, podKillerSelector, delayMillis);
     }
 
+    String twillRunnables = context.getConfigurations().get(TWILL_RUNNER_SERVICE_MONITOR_DISABLE);
```

Comment:
I think it would be a little better for this to be a boolean, and to set it to false in the SystemWorkerServiceLauncher. That way it's a bit more direct and also doesn't rely on kubernetes labeling details.

Reply:
if that works, should also remove 'kube' from the property name, as things in app-fabric aren't supposed to know about SPI implementations.

Reply:
So I initially did that. However, setting it to false in `SystemWorkerServiceLauncher` still returns default value. I believe it's because of https://github.com/cdapio/cdap/blob/develop/cdap-app-fabric/src/main/java/io/cdap/cdap/master/environment/k8s/MasterEnvironmentMain.java#L107 . Basically, it seems that we do not apply overriden values to cConf in MasterEnvironment.

Reply:
hm that's odd, what's the purpose of writing out those files then? They're added by SystemWorkerTwillApplication as files to localize, are we just ignoring them?

Could this cause other bugs?